src/common/random.hpp

```diff
@@ -11,5 +11,6 @@ void rnd_init(void);
 int32 rnd(void);// [0, SINT32_MAX]
 int32 rnd_value(int32 min, int32 max);// [min, max]
 bool rnd_chance( uint16 chance, uint16 base );
+bool rnd_chance32( int32 chance, int32 base );
```

Comment:
```suggestion
bool rnd_chance( uint32 chance, uint32 base );
```
C++ has function overloading.

Reply:
`rnd_chance` is using `rnd_value` (`int32 rnd_value(int32 min, int32 max);`) leaving it in int32 was less annoying :(

The compiler shows me a warning from `rnd_chance` in `skill.cpp` when using the overloading, so I renamed the function. Not sure how to fix the warning